protoc-gen-swagger/genswagger/template.go

```diff
@@ -625,6 +631,23 @@ func renderServices(services []*descriptor.Service, paths swaggerPathsObject, re
 
 				methProtoPath := protoPathIndex(reflect.TypeOf((*pbdescriptor.ServiceDescriptorProto)(nil)), "Method")
 				desc := ""
+				var responseSchema swaggerSchemaObject
+
+				if b.Response == nil || len(b.Response.FieldPath) == 0 {
```

Comment:
Add a comment that this is resolving the value of `response_body` in the `google.api.HttpRule`